tcms/kiwi_auth/views.py

```diff
@@ -22,6 +22,7 @@
 User = get_user_model()  # pylint: disable=invalid-name
 
 
+# pylint: disable=missing-permission-required
 class LoginViewWithCustomTemplate(views.LoginView):  # pylint: disable=missing-permission-required
```

Comment:
Also remove the disable comments from the class/function definitions in files where you put a global disable directive.